llama-index-core/llama_index/core/objects/base.py

```diff
@@ -105,6 +140,27 @@ def __init__(
         self._index = index
         self._object_node_mapping = object_node_mapping
 
+    @property
+    def index(self) -> BaseIndex:
+        """Index."""
+        return self._index
+
+    @property
+    def object_node_mapping(self) -> BaseObjectNodeMapping:
+        """Object node mapping."""
+        return self._object_node_mapping
+
+    @staticmethod
+    def get_object_mapping(objects: Sequence[OT]) -> BaseObjectNodeMapping:
```

Comment:
why is this a public static method? should probably just be `get_default_object_mapping` as a standalone function

Reply:
Yea true. It was originally a normal class method lol